examples/react-app/src/Avatars.tsx

```diff
@@ -0,0 +1,28 @@
+import { PlmgAvatar } from '@ducky/plumage-react';
+
+const styles = {
+  marginLeft: '-15px',
+};
+
+export default function Avatars() {
+  return (
+    <>
+      <h3>Avatars</h3>
+      <div style={{ display: 'flex' }}>
+        <PlmgAvatar
+          size={'medium'}
+          imageUrl="https://static.ducky.eco/icons/maskable_icon_192.png"
+        ></PlmgAvatar>
+        <PlmgAvatar style={styles} size={'medium'}></PlmgAvatar>
+        <PlmgAvatar style={styles} size={'medium'} userDeleted></PlmgAvatar>
+        <PlmgAvatar
+          size={'medium'}
+          style={styles}
+          imageUrl="https://static.ducky.eco/icons/maskable_icon_192.png"
+        ></PlmgAvatar>
+        <PlmgAvatar style={styles} size={'medium'}></PlmgAvatar>
+        <PlmgAvatar size={'medium'} style={styles} userDeleted></PlmgAvatar>
```

Comment:
How could we programatically define if a user is deleted rather than explicitly saying they are? Is there something we can check the existence of?

Reply:
I've used an empty user name as the condition. Again, I imagine that it is more likely that the object would contain a user_deleted key or timestamp - but I used the lack of user name as a proxy - as I suppose all personal information would be deleted.